development/prow-addons-ctrl-manager/README.md

```diff
@@ -29,7 +29,7 @@ To ignore the ProwJob, add the **prow.kyma-project.io/slack.skipReport** label u
 postsubmits:
   - name: post-master-kyma-gke-upgrade
     branches:
-    - master
+    - ^master$
```

Comment:
I believe that regex should be in quotes "".

Reply:
As we discussed in our meeting, the prow implementation seems to take it without quotes here.